action.yaml

```diff
@@ -86,16 +118,28 @@ inputs:
   no_cert:
     description: 'Makes the application not to use a certificate by disabling certificate lookup.'
     required: false
-  app_directory:
-    description: 'Relative path for the directory of the app (i.e. where `Dockerfile` and `docker-compose.yaml` files are located). This is the directory that is copied to the EC2 instance.  Default is the root of the repo.'
+    
+  # Terraform
+  targets:
+    description: 'A list of targets to create before the full stack creation. Example: `'
```

Comment:
@PhillypHenning can you create an issue to further refine this?  It'd be nice to abstract this from the user with things like:
- `aws_efs_only: true` - would skip ansible and apply `-target` for just the EFS
- `postgres_only: true`
- etc

Reply:
https://github.com/bitovi/github-actions-deploy-docker-to-ec2/issues/104